input/size.go

```diff
@@ -202,6 +202,18 @@ const (
 	// which will transition an incoming HTLC to the delay-and-claim state.
 	HtlcSuccessWeight = 703
 
+	// HtlcTimeoutWeightConfirmed is the weight of the HTLC timeout
+	// transaction which will transition an outgoing HTLC to the
+	// delay-and-claim state, for the confirmed HTLC outputs. It is 3 bytes
+	// larger because of the cdditional CSV check in the input script.
+	HtlcTimeoutWeightConfirmed = 666
```

Comment:
😈

Reply:
Perhaps we should just have this (and also the above similarly), be: 
```go
ConfirmedScriptOverhead = 3
HtlcTimeoutWeightConfirmed = HtlcTimeoutWeight + ConfirmedScriptOverhead
```

Eliminates the `+3` magic number in the diff, and also some of these constants.

Reply:
Yeah, that is better. Done